### ISSUE TITLE
How can we have a project on staging with no recipient address?

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-dapps-v2#3276]
[This project](https://staging.giveth.io/project/teach-music-to-children?tab=donations) has no recipient address. it tells me to switch to network... nothing because there are no networks for it.

I think we should make it somehow impossible to have projects listed if the project owner removes the recipient address

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdnorIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--03ea949acac74af321e646317457c322d8713493/image.png)

cc @divine-comedian

**Comment by divine-comedian** [comment]
Good question! Maybe @mohammadranjbarz can take a look?

**Comment by mohammadranjbarz** [comment]
@divine-comedian  @laurenluz 
We had a bug a few month ago that editing projects in project page in admin panel made recipient addresses gone, it might be from that time, so I think we don't have this problem anymore

@maryjaf knows about it

**Comment by maryjaf** [comment]
> @divine-comedian @laurenluz We had a bug a few month ago that editing projects in project page in admin panel made recipient addresses gone, it might be from that time, so I think we don't have this problem anymore
> 
> @maryjaf knows about it

Yeah, this issue has been resolved and the projects that have this problem should be edited again by project owner

**Comment by divine-comedian** [comment]
okay - closing this issue if we can confirm the root has been resolved.
